engine/configurator/conf_yaml.c

```diff
@@ -349,6 +525,13 @@ parse_config_yaml_dependency_entry(yaml_document_t *d,
                 return TE_EINVAL;
             }
         } while (++pair < n->data.mapping.pairs.top);
+
+        if (dep->oid == NULL)
+        {
+        ERROR(CS_YAML_ERR_PREFIX
+              "found the oid field is absent in 'depends' node");
+        return TE_EINVAL;
```

Comment:
Why is it wrong? This indentation helps do not break the string in quotes and still satisfy 80 chars restriction.

Reply:
Because the indentation shows the code structure, thus the rules  of indentation must never be violated. I see nothing wrong with splitting a string literal in two lines, it happens all the time, but I suspect that if you don't write "found", you fit the line.

Reply:
> Indentation level is 4 character. It means that every nested block shifted to 4 character right relatively to the  
> position of its “parent block”.

Reply:
I didn't find the rule for the case like
```
very_long_name_of_the_function(the_first_very_long_name_of_the_variable,
                                                      the_second_very_long_name_of_the_variable);
```
that is very common in the code. And I thought that this case is the same.

Reply:
No, how is it related? In your code `ERROR` and `return` were aligned under `{`, not indented 4 spaces to the left, as they should.

Reply:
Now I see. I missed that ERROR was misindented. And thought that you talking about the fact that I started `"found the oid field..."` on the next line.